docs/content/docs/build/audience.md

```diff
@@ -78,41 +79,40 @@ The `getMyself` method returns the `IMember` object from the audience correspond
 
 {{% callout tip %}}
 
-Connection transitions can result in short timing windows where `getMyself` returns undefined.  This is because the current client connection will not have been added to the audience yet, so a matching connection ID cannot be found.
+Connection transitions can result in short timing windows where `getMyself` returns undefined.  This is because the current client connection will not have been added to the audience yet, so a matching connection ID cannot be found.  Similarly, offline scenarios may produce the same behavior.
 
 {{% /callout %}}
 
-#### Events
+### Events
 
 #### membersChanged
 
-The `membersChanged` event fires whenever an externally visible change to the audience members is made.  Listeners may call the `getMembers` method to get the new list of members.
+The `membersChanged` event is emitted whenever an externally visible change to the audience members is made.  Listeners may call the `getMembers` method to get the new list of members.
 
 #### memberAdded
 
-The `memberAdded` event fires whenever an externally visible member or client connection is added to the audience.  The event also provides the connection client ID and the `IMember` object for this change.
+The `memberAdded` event is emitted whenever an externally visible member or client connection is added to the audience.  The event also provides the connection client ID and the `IMember` object for this change.
 
 #### memberRemoved
 
-The `memberRemoved` event fires whenver an externally visible member or client connection leaves the audience.  The event also provides the connection client ID and the `IMember` object for this change.
+The `memberRemoved` event is emitted whenver an externally visible member or client connection leaves the audience.  The event also provides the connection client ID and the `IMember` object for this change.
 
 ## Using Audience to Build Presence Features
 
 ### Data Management and Inter-User Communication
 
-While the audience provides the base for user presence features, the list of connected users does not provide a compelling experience on its own.  Your specific scenario will involve working with additional user data, which falls broadly into the following three categories.
+While the audience is the foundation for user presence features, the list of connected users does not provide a compelling experience on its own.  Your specific scenario will involve working with additional user data, which falls into a combination of the following three categories.
 
-#### Presence with No Shared Data
+#### Unshared Data
```

Comment:
```suggestion
#### Unshared data
```

I tend to think of this as "external and/or generated data". Is there a reason for the word "unshared?"

Reply:
it's to contrast with shared persisted and shared transient and form a set covering everything, because generated data could technically still need to be shared